index.html

```diff
@@ -307,22 +307,8 @@ <h3>BrowserCaptureMediaStreamTrack</h3>
               </li>
               <li>
                 <p>
-                  The user agent MUST validate <var>cropTarget</var> according to [=this=] track's
-                  current [=crop-state=].
-                </p>
-                <ul>
-                  <li>
-                    If [=this=] track is [=uncropped=], the user agent MUST only accept [=valid
-                    CropTargets=].
-                  </li>
-                  <li>
-                    If [=this=] track is [=cropped=], the user agent MUST accept either [=valid
-                    CropTargets=] or {{undefined}}.
-                  </li>
-                </ul>
-                <p>
-                  If the user agent does not accept <var>cropTarget</var>, return a {{Promise}}
-                  [=rejected=] with an {{UnknownError}}.
+                  If <var>cropTarget</var> is neither a [=valid CropTarget=] nor {{undefined}}, the
```

Comment:
The [argument](https://w3c.github.io/mediacapture-region/#dom-browsercapturemediastreamtrack-cropto) is nullable.
```suggestion
                  If <var>cropTarget</var> is neither a [=valid CropTarget=] nor {{null}}, the
```